File: apollo/src/main/java/io/muun/apollo/domain/errors/RecoveryCodeVerificationError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class RecoveryCodeVerificationError extends UserFacingError {


    public RecoveryCodeVerificationError() {
        super(UserFacingErrorMessages.INSTANCE.recoveryCodeVerification());
    }

    public RecoveryCodeVerificationError(Throwable cause) {
        super(UserFacingErrorMessages.INSTANCE.recoveryCodeVerification(), cause);
    }
}

File: apollo/src/main/java/io/muun/apollo/domain/errors/RecoveryCodeVerificationError.kt
package io.muun.apollo.domain.errors


import io.muun.apollo.external.UserFacingErrorMessages

class RecoveryCodeVerificationError: UserFacingError {

    constructor():
        super(UserFacingErrorMessages.INSTANCE.recoveryCodeVerification())

    constructor(cause: Throwable):
        super(UserFacingErrorMessages.INSTANCE.recoveryCodeVerification(), cause)

}
